src/frontend/src/lib/styles/global/gix.scss

```diff
@@ -115,9 +116,9 @@ div.input-field input[id]::placeholder {
 }
 
 div.modal {
-	--overlay-background: var(--color-white);
+	--overlay-background: #f8f9fa;
```

Comment:
I think you created `ghost-white` for this?

Reply:
Ah yeah good catch thanks. Totally forgot about it, I was so focus having to redo these freaking screenshots zillion times. Changed.